pyproject.toml

```diff
@@ -29,6 +29,7 @@ dev = [
   "pytest",
   "pytest-cov",
   "requests-mock",
+  "sentry-sdk[flask]",
```

Comment:
Why is sentry a dev dependency, it’s only ever used in production? :thinking:

Reply:
It’s needed to run the tests.